front-dispatcher/src/components/Header/FixedHeader/style.ts

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import styled from "styled-components";
+import { colors } from "../../../strings/colors";
+
+const HeaderStyle = styled.header`
+  position: fixed;
+  width: 100vw;
+  height: 10vh;
+  top: 0;
+  left: 0;
+  background-color: ${colors.Dark};
```

Comment:
I think you should give colors names with more logic like dark red, light black and not just dark